M2/Macaulay2/packages/CellularResolutions.m2

```diff
@@ -42,8 +42,8 @@ export {--types
         "CellDimension",
         "InferLabels",
         "LabelRing",
-        "Reduced",
```

Comment:
Pedantically speaking, removing the comma here is not necessary since `export` silently ignores null entries.